lhotse/recipes/xbmu_amdo31.py

```diff
@@ -0,0 +1,126 @@
+"""
+About the XBMU-AMDO31 corpus
+XBMU-AMDO31 is an open-source Amdo Tibetan speech corpus published by Northwest Minzu University.
```

Comment:
Could you add some more details about the corpus? It seems the HF link does not contain any description either. You can refer to the other recipes.

Reply:
Ok, HF will update today with more descriptions and add more details to this part of the code.